src/OrleansSQLUtils/Storage/RelationalStorage.cs

```diff
@@ -146,11 +146,7 @@ public Task<IEnumerable<TResult>> ReadAsync<TResult>(string query, Action<IDbCom
                 throw new ArgumentNullException("selector");
```

Comment:
Another quick plunge on the failing relational storage tests, here are the three first grain reference IDs from the first run:
1. {*grn/3/00000000+úÎhùBìóxWÚCôÃép}
2. {*grn/3/00000000+ðènûùlæøþáýøçüÁ}
3. {*grn/3/00000000+ðènûùlæøþáýøçüÁ}

The second and third ones are the same, so the test fails. This is taken from `StorageDataSetGenericHuge_Json_WriteRead`, which uses `StorageDataSetGeneric`, which in turn use `RandomUtilities.GetRandomGrainReference` go generate the reference. The thing is, these are deterministically not random now. I don't know the reason. Quickly spelunkig through the code, my hazard quess would be line `(Func<IInternalGrainFactory, GrainReference>)(grainFactory => RandomUtilities.GetRandomGrainReference<TGrainKey>(grainFactory)),` maybe return just being the same instance and hence returning the same value. This is just a hazard guess, but it's midnight and if someone has a fresh pair of eyes (@jdom, @ReubenBond?), it'd be great.